hibernate-core/src/test/java/org/hibernate/orm/test/querycache/PersonName.java

```diff
@@ -76,4 +76,13 @@ public int hashCode() {
 		result = 31 * result + ( last != null ? last.hashCode() : 0 );
 		return result;
 	}
+
+	@Override
+	public String toString() {
+		return "PersonName{" +
+				"first='" + first + '\'' +
+				", middle='" + middle + '\'' +
+				", last='" + last + '\'' +
+				'}';
+	}
```

Comment:
Is the above code change to `PersonName` necessary for this PR? Seems it was not used in the testing case.

Reply:
reverted, thanks @NathanQingyangXu

Reply:
:shrug: often these kinds of things are useful in debugger